### DISCUSSION TITLE
🤔 如何设置饼图的原点坐标

### DISCUSSION THREAD

**Discussion by BlucasWu** [antvis/G2Plot#3585] [Q&A]
<img width="414" alt="图片" src="https://user-images.githubusercontent.com/126557877/221799351-1cd0a1e6-322a-4e05-a32b-170317536ca2.png">

原始需求是这三个饼图是分别对应一个 Pie 实例，想让这三个饼图左对齐 @visiky

**Comment by cdx111** [comment]
I solved the same problem by setting padding property

**Comment by hustcc** [comment]
设置 padding 可以解决这个问题，但是不是特别优雅，G2 4.x 中不是特别好解决这个问题，我们会在 G2 5.0 中考虑进去。
